scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -324,6 +324,7 @@ object Type {
 
 @branch trait Pat extends Tree
 object Pat {
+  @ast class ArgClause(values: List[Pat]) extends Member.ArgClause
```

Comment:
just like with Term.ArgClause (which is a Tree but not a Term), this one is also not a Pat, because it cannot be used as a pattern by itself.

there's also another, a bit more obscure reason: we have implicits converting between List[Term] and Term.ArgClause, and if Term.ArgClause was declared a Term, then some invalid quasiquotes would suddenly start working, because Term derives from Stat and hence anything that can be implicitly converted to a Term would parse as a Stat.

Reply:
Ach ok, nvm then!